### DISCUSSION TITLE
Should drag and dropping a task created in "today" to a project move it to the project?

### DISCUSSION THREAD

**Discussion by rommeswi** [super-productivity/super-productivity#3155] [Q&A] [answered]
I created a task and later realized I want to add it to a project. I thought it was natural to just drag and drop it onto the project but that does not work. Is this a bug? Or is this not intended?

I just added it via the menu, which is fine, just not as natural.

**Comment by johannesjo** [comment, marked as answer]
Please refer to #810. You can also use the "move to project" shortcut. That's how I do it personally :)
